src/pages/leaderboard.tsx

```diff
@@ -1,16 +1,15 @@
 import { useCallback, useEffect, useRef, useState } from "react";
 import { useSession } from "next-auth/react";
 import { Box, Typography } from "@mui/material";
-
 import { type LeaderboardData } from "~/types/index.js";
 import { isValidSession } from "~/utils/session.js";
 import { AddScoreTable, LeaderboardTable } from "~/components/tables/index.js";
 import { leaderboardData } from "~/constants/index.js";
 import { text } from "~/assets/text/index.js";
-import { styles } from "./styles.js";
+import { styles } from "../styles/pages/leaderboard.js";
```

Comment:
Should we use "~" for referencing the path?

Reply:
good one, they should be good now. wondering if I can configure the automatic imports feature on vscode to use absolute paths